docs/partner_editable/additional_info.adoc

```diff
@@ -83,20 +85,22 @@ If you have other weather dataset you want to use, just make sure it contains at
 |Type
 |Mandatory
 |Format
-|time |string|X|  yyyy-MM-dd HH:mm:ss
+|time|string|X|  yyyy-MM-dd HH:mm:ss
 |temperature| double|X|
-|apparenttemperature |double|X|
-|humidity |double|X|
+|apparenttemperature|double|X|
+|humidity|double|X|
 |===
 
-The CloudFormation parameter "WithWeather" should have been set to 1 to indicate that weather data is available to use, for generating ML model.
+The CloudFormation parameter `WithWeather` should have been set to 1 to indicate that weather data is available to use for generating the ML model.
+
+=== Prepare geolocation data
 
-=== Prepare geo-location data
+This Quick Start requires geolocation data to be uploaded to the `geodata` folder in the business-zone S3 bucket. You can get the name of the this bucket from the output section of the CloudFormation stack. The S3 path of geodata location should be `s3://<business-zone-s3-bucket-name>/geodata/`. The data should be in a .CSV file format with the following structure, where all three columns are mandatory with comma-separated values.
 
-This Quick Start requires geo-location data to be uploaded to the _geodata_ folder in Business Zone S3 bucket. You can get the name of the Business zone S3 bucket from the output section of the CloudFormation stack. The S3 path of geo-data location should be _s3://<business-zone-s3-bucket-name>/geodata/_. The data should be in a .CSV file format with the following structure, where all 3 columns are mandatory with comma-separated values.
+//TODO Shivansh, we say "all three columns," but there are only two.
```

Comment:
Each field is a column in the csv file, hence 3 columns.

Reply:
Aha. I'll change this to "all three fields."